composer.json

```diff
@@ -19,12 +19,12 @@
     "require": {
         "php": "^7.1",
         "psy/psysh": "^0.8 || ^0.9",
-        "symfony/expression-language": "^3.4 || ^4.0",
-        "symfony/framework-bundle": "^3.4 || ^4.0"
+        "symfony/expression-language": "^4.3",
+        "symfony/framework-bundle": "^4.3"
     },
     "require-dev": {
         "phpunit/phpunit": "^6.4",
```

Comment:
6.4... time sure flies huh. I think we can upgrade that dependency but that can be done in another PR I guess

Reply:
Agreed, created an issue: https://github.com/theofidry/PsyshBundle/issues/45